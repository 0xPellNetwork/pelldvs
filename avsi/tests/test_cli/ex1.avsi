echo hello
info
query "abc"
query "def"
request keykeykey valuevalue