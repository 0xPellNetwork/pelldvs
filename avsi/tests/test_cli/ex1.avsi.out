> echo hello
-> code: OK
-> data: hello
-> data.hex: 0x68656C6C6F

> info 
-> code: OK

> query "abc"
-> code: OK
-> height: 0
-> key: abc
-> key.hex: 616263

> query "def"
-> code: OK
-> height: 0
-> key: def
-> key.hex: 646566

> request keykeykey valuevalue
-> code: OK
-> data: keykeykey
-> data.hex: 0x6B65796B65796B6579

